examples/tool-use-vercel/src/financial-analysis.service.ts

```diff
@@ -177,14 +181,7 @@ export class FinancialAnalysisService {
     this.sessionId = sessionId;
     this.windowId = windowId;
 
-    const goal = `Extract the current price of NVDA and the growth percentage over all the time periods available. 
-    There may be buttons on the page to access historical information.
-    Use Google Finance to access stock data at https://www.google.com/finance/quote/NVDA:NASDAQ`;
-
-    const { text } = await this.sendPrompt({ goal });
-
-    this.log.info(`Received result: ${text}`);
```

Comment:
Do you still want to log this? looks like you removed the log

Reply:
Yeah, that was intentional. I was double logging because we log when the cli is done.